src/django/api/serializers.py

```diff
@@ -615,6 +677,31 @@ def get_activity_reports(self, facility):
         return FacilityActivityReportSerializer(
             facility.activity_reports(), many=True).data
 
+    def get_contributor_fields(self, facility):
+        request = self.context.get('request') \
+            if self.context is not None else []
+        if request is None or request.query_params is None:
+            return []
+
+        embed = request.query_params.get('embed')
+        contributor_id = request.query_params.get('contributor', None)
+        if not embed == '1' or contributor_id is None:
+            return []
+
+        contributor = Contributor.objects.get(id=contributor_id)
+        if contributor.embed_config is None:
+            return []
+
+        config = contributor.embed_config
+        fields = EmbedField.objects.filter(
+            embed_config=config, visible=True).order_by('order')
+
+        list_items = FacilityListItem.objects.filter(
+                facility=facility,
+                source__contributor=contributor)
```

Comment:
Not captured in the original issue description is the need to allow contributors to dissociate from facilities they no longer do business with, which we allow for via the `is_active` flags. I expect that adding 2 additional filter statements to this call to `.filter()` should take care of this

```
source__is_active=True, 
facilitymatch__is_active=True
```

Although the same facility could be submitted multiple times and therefore have multiple line items matched to it, I think that we should assume that the most recent, active submission is the one from which we should pull the nonstandard field values. I expect that adding an `order_by('-created_at')` and then taking just the first matching row will take care of this.